### ISSUE TITLE
R minimal set of vignettes are complete

### ISSUE THREAD

**Issue by pablo-gar** [chanzuckerberg/cell-census#248]
The following is considered a minimal set of vignettes, mirroring the following python notebooks

- [ ][census_axis_query.ipynb](https://github.com/chanzuckerberg/cell-census/blob/main/api/python/notebooks/api_demo/census_axis_query.ipynb)
- [ ] [census_dataset_presence.ipynb](https://github.com/chanzuckerberg/cell-census/blob/main/api/python/notebooks/api_demo/census_dataset_presence.ipynb)
- [ ] [census_dataset_presence.ipynb](https://github.com/chanzuckerberg/cell-census/blob/main/api/python/notebooks/api_demo/census_dataset_presence.ipynb)
- [ ] [census_query_extract.ipynb](https://github.com/chanzuckerberg/cell-census/blob/main/api/python/notebooks/api_demo/census_query_extract.ipynb)
- [ ] [comp_bio_census_info.ipynb](https://github.com/chanzuckerberg/cell-census/blob/main/api/python/notebooks/analysis_demo/comp_bio_census_info.ipynb)
- [ ] [comp_bio_query_data_and_metadata.ipynb](https://github.com/chanzuckerberg/cell-census/blob/main/api/python/notebooks/analysis_demo/comp_bio_query_data_and_metadata.ipynb)

**Comment by mlin** [comment]
Apr 24 status: 

* comp_bio_census_info is written, but blocked on a [tiledbsoma + TileDB Core issue](https://github.com/single-cell-data/TileDB-SOMA/issues/1262)
* census_data_integration (nee comp_bio_data_integration_scvi) [is also written](https://github.com/chanzuckerberg/cellxgene-census/pull/406) but not runnable in the limited RAM of GitHub Actions workers; @pablo-gar and I need to decide what to do about that
* the remaining two (census_compute_over_X and comp_bio_normalizing_full_gene_sequencing) are nice-to-haves IMO; they weren't originally on the list, and added after I suggested we do some of the fancier ones to make sure we touch all major parts of the tiledbsoma API, but I think the census_data_integration accomplishes that.
